### ISSUE TITLE
Request: Group items per feed

### ISSUE THREAD

**Issue by knireis** [FreshRSS/FreshRSS#3192]
New to FreshRSS, coming from from TinyRSS, and i like it sofar. However there is 1 thing which TTRSS has and i miss it on FreshRSS.
An option to show the items grouped per feed. So in my case first grouped per feed and inside the group on oldest first. The sequence of the groups is not important, might be alphabetical.
Is that possible?

**Comment by aledeg** [comment]
Maybe the global view is something you are looking for.

**Comment by knireis** [comment]
That seems a lot of clicking between threads. 
Would be nice if there is an option to have the posts displayed that way in the read view.

**Comment by pcause** [comment]
new to freshrss and looks very promising. I have categories and many feed within a category. When I view at the category level they are displayed by date but if the feed is within the category is a discussion the different items can be widely scattered and hard to follow.  I'd like to also ask for group by feed and within the feed sort by date (oldest or newest)

Thanks

**Comment by pcause** [comment]
question:  haven't found the db code yet, but you must be doing a select/sort to get the items, so can't you add an option to group in category and then optionally add a "group by" to the select??  I am sure it is a bit more complicated, but i'd assume the database does most of the work.

**Comment by davidfdriscoll** [comment]
I'd like this feature as well.

**Comment by pcause** [comment]
any thought from the devs on this.  if you have a feed that comes from a discussion board, it is very hard to follow without the ability to group by feed and then sort oldest/newest

**Comment by Alkarex** [comment]
We not not have the ability right now to sort by custom criteria (but that might come), but grouping by feed and then oldest/newest is natively available by picking the wanted feed in the left-hand menu

**Comment by pcause** [comment]
Thanks for the reply.    I'll try again.  I haven't used in a while because of this.  When I want to display a category do you pass SimplePie an array of feed URLs?  Seems that would sort results from all feeds in the category together by date.  If you fetched one at a time, you'd have the behavior I suggest, since each feed would be sorted by date but no intermixing of feed content.  Would this be a huge performance penalty??  I'd certainly pay the penalty.  But, again, thanks

**Comment by Frenzie** [comment]
SimplePie is only used for retrieval. Anything that's displayed is stored in the database.

**Comment by Warden20** [comment]
I'd like this feature as well.

**Comment by nclm** [comment]
Same as #2746 possibly? I’d be very interested in that feature too.
